api/AddComponent.go

```diff
@@ -37,7 +37,7 @@ func (api *API) AddComponent(projectName, componentName string) error {
 	log.Printf("adding component %s to project %s", componentName, projectName)
 	err = utils.AddKustomizeComponent(
 		nsPath,
-		filepath.Join(constants.ComponentRelPath, componentName),
+		[]string{string(filepath.Join(constants.ComponentRelPath, componentName))},
```

Comment:
type conversion here is redundant: 
`string(filepath.Join(constants.ComponentRelPath, componentName))` 
is the same as: 
`filepath.Join(constants.ComponentRelPath, componentName)`